### ISSUE TITLE
Feature request: Add support for displaying in satoshis

### ISSUE THREAD

**Issue by kaloudis** [ElementsProject/lightning-charge#74]
mBTC isn't very commonly used and sats are becoming somewhat of a standard. It would be awesome if lightning-charge displayed units in satoshis, especially when used on a paywall like [wordpress-lightning-publisher](https://github.com/ElementsProject/wordpress-lightning-publisher)

**Comment by kaloudis** [comment]
I'm happy to implement this myself, but would like to get some discussion going first on how this should function.

**Comment by shesek** [comment]
Implementing this was pretty straightforward so I went ahead and [made the BTC display unit configurable](https://github.com/ElementsProject/lightning-charge/commit/8ceae7a4a4947d0d3d81cafd3457a4059840d043) using a new `metadata.checkout.display_unit` field. It can be set to one of `msat`, `sat`, `bit`, `milli` or `btc`. For example:

```bash
$ curl $CHARGE/invoice -d msatoshi=150000 -d metadata[checkout][display_unit]=bit
```

Or with json:
```bash
$ curl $CHARGE/invoice -d msatoshi=150000 -H 'Content-Type: application/json' -d metadata='{"checkout":{"display_unit":"bit"}}'
```

I also changed the default display unit to satoshis.
